exercises/perfect-numbers/perfect_numbers_test.rb

```diff
@@ -0,0 +1,44 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'example'
+
+class PerfectNumberTest < Minitest::Test
+  def test_aliquot_sum_zero
+    assert_equal 0, PerfectNumber.new(1).aliquot_sum
```

Comment:
I don't know if we should force people to implement a public method called `aliquot_sum`. The exercise specifies that we need to classify numbers, how to do it should be open to the user. Should we test `aliquot_sum` or just a few cases of `classify`?